interface/src/app/node/node.js

```diff
@@ -4,7 +4,9 @@
 
 angular.module('adage.node', [
   'ui.router',
-  'ui.bootstrap'
+  'ui.bootstrap',
+
```

Comment:
Mmm, interesting. I do like to keep them a bit separate, so I will put in a comment here (since I do see why it would be strange to have just a blank line inside an array definition).